cloud-controller-manager/do/cloud.go

```diff
@@ -158,6 +162,13 @@ func newCloud() (cloudprovider.Interface, error) {
 		addr = fmt.Sprintf("%s:%s", addrHost, addrPort)
 	}
 
+	// var ipFamilies string
+	ipf, set := os.LookupEnv(doIPAddrFamiliesEnv)
+	ipFamilies = ipf
+	if set && !containsString(ipFamilyOpts, ipFamilies) {
```

Comment:
I'd rather prefer pulling the string split we do further down the stack up here. This allows us to work with a slice of tokenized family IDs in the business logic and do all the parsing in the bootstrap logic.

Let's also trim white spaces from split items.